provisioning/packer/http/preseed.cfg

```diff
@@ -29,22 +9,37 @@ d-i partman-auto-lvm/guided_size string max
 d-i partman-auto/choose_recipe select atomic
 d-i partman-auto/method string lvm
 d-i partman-lvm/confirm boolean true
+d-i partman-lvm/confirm boolean true
```

Comment:
@klonos thanks this is the source -- https://github.com/geerlingguy/packer-ubuntu-1604/blob/master/http/preseed.cfg#L11-L12

& it was fixed in a recent commit.